templates/sysv/lsb-3.1/init.sh

```diff
@@ -88,7 +88,13 @@ stop() {
       sleep 1
     done
     if status ; then
-      emit "$name stop failed; still running."
+      if [ "$KILL_ON_STOP_TIMEOUT" -eq 1 ] ; then
+        trace "Timeout reached. Killing $name (pid $pid) with SIGKILL.  This may result in data loss."
```

Comment:
I think it's ok as one line. We can see how users feel and adapt it as complaints occur.